docs/puzzles/day7.md

```diff
@@ -2,12 +2,321 @@
 sidebar_position: 7
 ---
 
-# Day 7: Custom Customs
+import Solver from "../../../../website/src/components/Solver.js"
+
+# Day 7: The Treachery of Whales
+
+by [@tgodzik](https://github.com/tgodzik)
 
 ## Puzzle description
 
 https://adventofcode.com/2021/day/7
 
+## Part 1: Fast and craby
+
+The whale is trying to swallow our submarine! Fortunately, there is a small
+armada of crabs, let's call it a crabmada, in their tiny crabmarines that want
+to help us out! However, the crabs are not particularly efficient and will need
+our help, we need to align all their little crabmarines using the least fuel
+possible, so that they can blow a whole in the ocean floor.
```

Comment:
```suggestion
possible, so that they can blow a whale in the ocean floor.
```

Reply:
I think it should be `hole`, actually ;)

Reply:
Fixed!